### PR TITLE
added new ways to ask for state

### PR DESCRIPTION
**Pull request by fejrr**

### CODE REVIEW

**Comment by Uriziel01** [comment]
Will have a look at this in the evening

**Comment by fejrr** [comment]
which evening :)

**Comment by Uriziel01** [comment]
Sorry @fejrr got a bit distracted 😄 Will have a look tomorrow.

File: sentences/pl/homeassistant_HassGetState.yaml [hunk 1]
```diff
@@ -18,6 +18,14 @@ intents:
...
+      - sentences:
+          - stan <name> [<area>]
+        response: one_state
+        excludes_context:
+          domain:
+            - scene
+            - script
+
       - sentences:
           - jak(i|a|e) jest status <name> [<area>]
```
Comment (Uriziel01) [inline sentences/pl/homeassistant_HassGetState.yaml:30]: Can we please also fix the `jake` issue I can see here, please adapt this to `(i|a|ie)`

File: sentences/pl/homeassistant_HassGetState.yaml [hunk 2]
```diff
@@ -26,6 +34,14 @@ intents:
...
+      - sentences:
+          - status <name> [<area>]
+        response: one_status
+        excludes_context:
+          domain:
+            - scene
+            - script
+
       - sentences:
           - jak(i|a|e) jest wartość <name> [<area>]
```
Comment (Uriziel01) [inline sentences/pl/homeassistant_HassGetState.yaml:46]: Also here

File: sentences/pl/homeassistant_HassGetState.yaml (hunk 1 above)
Comment (Uriziel01) [inline sentences/pl/homeassistant_HassGetState.yaml:30]: Could this please cover both `status` and `stan` at the same time?

File: sentences/pl/homeassistant_HassGetState.yaml [hunk 3]
```diff
@@ -18,6 +18,14 @@ intents:
             - scene
             - script
 
+      - sentences:
+          - stan <name> [<area>]
```
Comment (Uriziel01) [inline sentences/pl/homeassistant_HassGetState.yaml:22]: Could this please cover both `status` and `stan` at the same time?

**Review by Uriziel01 — CHANGES_REQUESTED** [review]

File: sentences/pl/homeassistant_HassGetState.yaml (hunk 1 above)
Comment (fejrr) [reply sentences/pl/homeassistant_HassGetState.yaml]: also removed jaka since noone talks like that

Comment (fejrr) [reply sentences/pl/homeassistant_HassGetState.yaml]: don't know if i did this right, I just grouped sentences

**Review by Uriziel01 — APPROVED** [review]

**Pull request merged**